action.yml

```diff
@@ -9,6 +9,7 @@ inputs:
   repo-token:
     description: "Token with permissions to do repo things"
     required: false
+    default: "${{ github.token }}"
```

Comment:
Could be replaced with `default: ${{ github.server_url == 'https://github.com' && github.token || '' }}` as in [actions/setup-go](https://github.com/actions/setup-go/blob/main/action.yml).